.travis.yml

```diff
@@ -18,35 +18,35 @@ branches:
 matrix:
   include:
     - os: linux
-      dist: trusty
+      dist: xenial
       sudo: required
-      env: SWIFT_SNAPSHOT=4.0.3
+      env: DOCKER_IMAGE=swift:4.0.3 SWIFT_SNAPSHOT=4.0.3
```

Comment:
Doesn't this need `services: docker`?

Reply:
Good spot!  I'm surprised it worked anyway